messaging/src/test/java/org/axonframework/messaging/StubProcessingContext.java

```diff
@@ -17,6 +17,7 @@
 package org.axonframework.messaging;
 
 import jakarta.annotation.Nonnull;
+import org.axonframework.messaging.unitofwork.LegacyUnitOfWork;
```

Comment:
Another check, just as with the `LegacyMessageSupportingProcessingContext`:

- The `StubProcessingContext` will be used by **all** infrastructure components under test **if** they do not use the "full" `UnitOfWork`.


Is that correct?